README.md

```diff
@@ -36,91 +36,47 @@ java -jar app-v-0.1.0.war
-- `onto-viewer/docker/web/nuxt.config.js` > `onto-viewer-web/nuxt.config.js`
-- `onto-viewer/docker/web/Dockerfile` > `onto-viewer-web/Dockerfile`
-4. In the `onto-viewer/docker/runtime/server/` folder, you must put configuration and ontologies files. You can find samples of these files in the `onto-viewer-config-loader/src/main/resources`. Note that the `onto-viewer/docker/runtime/ `folder is excluded from Git, so you can freely put there any file you want.
-The final folder system should look like shown below:
-
-```
-onto-viewer
-|   .gitignore
-|   CHANGELOG.md
-|   CODE_OF_CONDUCT.md
-|   CONTRIBUTING.md
-|   DCO
-|   docker-compose.yaml
-|   LICENSE
-|   pom.xml
-|   README.md   
-|   api-doc/
-|   docker/
-|   |   init-onto-viewer.sh
-|   runtime/
-|   |   server/ 
-|   |       config <- config files go here    
-|   |       ontologies <- ontologies go here
-|   server
-|   |   Dockerfile      
-|   web
-|   |   nuxt.config.js         
-|   onto-viewer-config-loader
-|   onto-viewer-core
-|   onto-viewer-toolkit
-|   onto-viewer-web-app 
-|   style          
-onto-viewer-web <- installed and configured general tempaltes from general directory
-|   .dockerignore
-|   ...
-|   Dockerfile
-|   jsconfig.json
-|   nuxt.config.js
-|   ...
-|   api
-|	...
-|   store
-|   strapi
-strapi-dashboard <- installed and configured Strapi
-|   .dockerignore
-|	...
-|   Dockerfile
-|   favicon.png
-|   package-lock.json
-|   package.json
-|   README.md
-|   yarn.lock
-|   .tmp/   
-|   config/
-|   database/
-|   public/
-|   src/
-```
-  
+ In the `onto-viewer/docker/runtime/server/` folder, you must put configuration and ontologies files. You can find samples of these files in the `onto-viewer-config-loader/src/main/resources`. Note that the `onto-viewer/docker/runtime/ `folder is excluded from Git, so you can freely put there any file you want.
 
 Then, from the `onto-viewer/` folder run the following command to start the applications:
 
 ```
 docker-compose up --build -d
 ```
-To list applications that have started, use ```docker-compose ps```.
 
 Please note that it takes a while to for all services to start depending on how many ontologies you provided.
 
 After all ontologies are loaded, the Onto viewer will be accessible from http://localhost:3000/dev/ontology. 
 
+
+You can see all running containers and their status:
+
+```
+$ docker compose ps
+
+NAME                   IMAGE                COMMAND                  SERVICE             CREATED             STATUS              PORTS
+onto-viewer-server-1   onto-viewer-server   "docker-entrypoint.s…"   onto-viewer-server  20 seconds ago      Up 17 seconds       0.0.0.0:8080->6101/tcp
+web-with-strapi-1      web-with-strapi      "docker-entrypoint.s…"   web-with-strapi     4 hours ago         Up 17 seconds       0.0.0.0:1337->1337/tcp
+```
+
+If you want to see logs from one container use:
+
+```
+# to view continuous log output
+$ docker logs --follow <container name>
```

Comment:
```suggestion
$ docker compose logs --follow <container name> # where "<container name>" is "onto-viewer-server" or "web-with-strapi"
```

Reply:
@przemekgradzki Shouldn't be `docker-compose`?